website/docusaurus.config.js

```diff
@@ -196,8 +196,8 @@ const siteConfig = {
         },
       ],
     },
-
     algolia: {
+      appId: "6DNGPTCB04",
```

Comment:
This is a place holder, still waiting for the actual appId from the algolia team.

Reply:
will need to transfer app ownership later on too

Reply:
yup, but currently no update from algolia team.